# Geronimo Dubra

ðŸ“§ geronimo.dubra@gmail.com    
ðŸ”— [LinkedIn](https://linkedin.com/in/geronimo-dubra/)  

---

## Profile
Hi! I'm Geronimo, a trained bioinformatician with over 12 years of experience in biological data science within academic settings. After transitioning to the private sector, I had the opportunity to work on machine learning projects, which I am eager to continue exploring. I am particularly interested in computer vision and the application of agentic solutions based on small LLMs deployed on-premise. On a personal level, I am a critical thinker with strong interpersonal skills and truly enjoy collaborative work.

---

## Education
- **PhD in Biology and Health** â€“ UniversitÃ© de Montpellier, France  
  Thesis defense in September 2022

---

## Professional Experience

### Data Consultant â€“ R&D Data Scientist  
**Seenovate â€“ Montpellier** | 09/2024 â€“ 04/2025  
- Led R&D efforts to design and implement workflows using SHAP values to enhance machine learning performance:  
  - Developed a CNN for biomedical image classification.  
  - Evaluated model explainability and interpretability.  
  - Prototyped active learning workflows using SHAP values.  
- Initiated and established a commercial partnership with MyCenter Palavas.  
- Developed a Python API to control and test a Raspberry Pi camera system.  
- Created educational material for a course on biostatistics.  

### PhD Student / Postdoctoral Researcher  
**Daniel Fisherâ€™s Group â€“ IGMM, Montpellier** | 05/2018 â€“ 01/2024  
- Built bioinformatics pipelines for high-throughput sequencing data analysis.  
- Provided statistical consulting and support for data analysis and visualization.  
- **PhD Project:** Investigated the regulation of the cell cycle through phosphorylation of disordered proteins.  
- **Postdoc Project:** Performed bioinformatics analysis on the evolutionary conservation of Ki-67 using deep learning tools.  

### Bioinformatics Intern  
**Krzysztof Rogowskiâ€™s Group â€“ IGH, Montpellier** | 03/2017 â€“ 12/2017  
- Conducted statistical phylogenetic and structural biology analyses.  
- Developed scripts to automate sequence profile searches and in-silico mutagenesis analyses.  

### Research Assistant  
**Argentine Bioinformatics Platform â€“ Buenos Aires** | 09/2014 â€“ 04/2018  
- Analyzed exome sequencing data to identify disease-causing mutations in rare genetic disorders.  
- Mined genomic databases to extract relevant data.  
- Developed machine learningâ€“based tools to predict the clinical relevance of genetic mutations.  


## Technical Skills

- **Programming:** Python, R, Bash, C#  
- **Data Science & ML:** pandas, scikit-learn, PyTorch, XGBoost, SHAP  
- **APIs & Web Dev:** FastAPI, Flask, CSS, HTML  
- **Databases & Data Management:** SQL, Neo4J  
- **Visualization:** matplotlib, seaborn, plotly, ggplot2, shiny  
- **DevOps & Tools:** Git, Docker, Conda, Jupyter, uv, MLflow  


---

## Additional Courses
- Specialization in Machine Learning â€“ DeepLearning.AI & Stanford University (Coursera) | Feb 2023  
- ChIP-seq, RNA-seq and Hi-C: Data Processing & Visualization â€“ IGMM Montpellier | Jan 2020  
- Introduction to Analytical Approaches for Systems Biology and Bioinformatics â€“ Univ. of Montpellier | Mar 2019  
- Scientific Integrity in Research Professions (MOOC) | Jan 2019  
- Industrial Biotechnology â€“ Univ. of Buenos Aires | Mar 2018  
- Genomics and Human Population Diseases â€“ Univ. of Buenos Aires | Sept 2015  
- The Data Scientistâ€™s Toolbox â€“ Johns Hopkins (Coursera) | Jun 2015  
- Computing for Data Analysis â€“ Johns Hopkins (Coursera) | Nov 2012  


---

## Publications (Selected)
- Valverde J.M., *Dubra, G.*, et al. (2023). A cyclin-dependent kinase-mediated phosphorylation switch of disordered protein condensation. *Nat Commun*, 14, 6316. [DOI](https://doi.org/10.1038/s41467-023-42049-0)  
- Prieto, S., *Dubra, G.*, Camasses, et al. (2023). CDK8 and CDK19 act redundantly to control the CFTR pathway in the intestinal epithelium. *EMBO Reports*, 24(2). [DOI](https://doi.org/10.15252/embr.202154261)  
- *Dubra, G.* (2022). A molecular mechanism for the quantitative model of the cell cycle: CDK-mediated phosphorylation of intrinsically disordered region.  
- Mrouj, K., AndrÃ©s-SÃ¡nchez, N., *Dubra, G.*, et al. (2021). Ki-67 regulates global gene expression and promotes sequential stages of carcinogenesis. *PNAS*, 118(10). [DOI](https://doi.org/10.1073/pnas.2026507118)  
- van der Laan, S., LÃ©vÃªque, et al. (2019). Evolutionary Divergence of Enzymatic Mechanisms for Tubulin Detyrosination. *Cell Reports*, 29(12). [DOI](https://doi.org/10.1016/j.celrep.2019.11.074)  
- Ma, C. A., et al. (2017). Germline hypomorphic CARD11 mutations in severe atopic disease. *Nature Genetics*, 49(8), 1192â€“1201. [DOI](https://doi.org/10.1038/ng.3898)  

---


